lib/cloopen/voice.rb

```diff
@@ -0,0 +1,37 @@
+require 'rest_client'
+require 'uri'
+require 'json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.